### PR TITLE
Import Dart rules from buildroot, to make them available in Fuchsia.

### PR DESCRIPTION
**Pull request by pylaligand**
This also removes a reference to mojo_gni.

### CODE REVIEW

File: build/dart_rules.gni [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    if (defined(invoker.sdk_ext_files)) {
+      sdk_ext_files += invoker.sdk_ext_files
+    }
+
+    sdk_ext_mappings = []
+    if (defined(invoker.sdk_ext_mappings)) {
+      sdk_ext_mappings += invoker.sdk_ext_mappings
+    }
+
+    script = rebase_path("build/dart/dart_pkg.py", ".", "//")
```
Comment (pylaligand) [inline build/dart_rules.gni]: The only changes in this file are the removal of the mojo.gni import and the use of mojo_sdk_root to rebase various paths (now replaced by "//').

**Comment by pylaligand** [comment]
Never mind, I forgot to include some necessary files in this change.

**Comment by pylaligand** [comment]
Good to go.

**Comment by abarth** [comment]
LGTM

**Pull request merged**